[EDU-3657] feat: add managed config (#555)
aziontech/docs: src/content/docs/pt-br/pages/menu-principal/referencia/servicos/managed-configurations.mdx

---
title: Managed Configurations

> description: O serviço de Managed Configurations oferece gerenciamento e controle sobre as configurações da sua infraestrutura edge da Azion. Ao adquirir este serviço, uma equipe de engenheiros experientes pode realizar certas tarefas diárias em seu nome, como configurar, monitorar e manter as configurações para você.

Reviewer: ```suggestion
description: Ao adquirir este serviço, uma equipe de engenheiros experientes pode realizar certas tarefas diárias em seu nome, como configurar, monitorar e manter as configurações para você.
```

> meta_tags: serviços, Managed Configurations, configurações, configurações

Reviewer 2: ```suggestion
meta_tags: serviços, Managed Configurations, configurações
```

namespace: documentation_services_managed_configuration
permalink: /documentation/services/managed-configurations/
---

O serviço de **Managed Configurations** oferece gerenciamento e controle sobre as configurações da sua infraestrutura edge na Azion. Ao adquirir este serviço, uma equipe de engenheiros experientes pode realizar certas tarefas diárias em seu nome, como configurar, monitorar e manter as configurações.

O serviço de Managed Configurations é operado pela equipe de Arquitetos de Soluções da Azion, que será responsável por implementar as alterações de configuração solicitadas pelo cliente da Azion. Para solicitar o serviço de Managed Configurations para aplicações rodando na Azion, [abra um Ticket de Suporte](https://tickets.azion.com/).

:::note

> Este add-on está disponível para os Planos de Serviço **Enterprise** e **Mission Critical**. Este serviço tem um pacote mensal com um preço fixo, limitado por solicitação de configuração. Cada solicitação de serviço gera uma alteração de configuração. Consulte [Preços](https://www.azion.com/pt-br/precos/) para mais informações sobre como este serviço é cobrado.

Author: ```suggestion
Este add-on está disponível para os Planos de Serviço **Enterprise** e **Mission Critical**. Este serviço tem um pacote mensal com um preço fixo, limitado por solicitação de configuração. Cada solicitação do serviço dá direito a uma alteração de configuração. Consulte [Preços](https://www.azion.com/pt-br/precos/) para mais informações sobre como este serviço é cobrado.
```

> Este add-on está disponível para os Planos de Serviço **Enterprise** e **Mission Critical**. Este serviço tem um pacote mensal com um preço fixo, limitado por solicitação de configuração. Cada solicitação de serviço gera uma alteração de configuração. Consulte [Preços](https://www.azion.com/pt-br/precos/) para mais informações sobre como este serviço é cobrado.

Reviewer 2: mesmo comment de EN, não ficou tão claro

:::

## Escopo

> A tabela abaixo descreve os produtos e capabilities inclusos, bem como as principais configurações no escopo das **Managed Configurations**.

Author: ```suggestion
A tabela abaixo descreve os produtos e funcionalidades inclusos, bem como as principais configurações no escopo das **Managed Configurations**.
```

> 
> ## Escopo

Reviewer: ```suggestion

---

## Escopo
```

> A tabela abaixo descreve os produtos e capabilities inclusos, bem como as principais configurações no escopo das **Managed Configurations**.

Reviewer 2: ```suggestion
A tabela abaixo descreve os produtos e capacidades inclusos, bem como as principais configurações no escopo das **Managed Configurations**.
```

> A tabela abaixo descreve os produtos e capabilities inclusos, bem como as principais configurações no escopo das **Managed Configurations**.

Reviewer: ```suggestion
A tabela abaixo descreve os produtos e recursos inclusos, bem como as principais configurações no escopo das **Managed Configurations**.
```

> | Produtos e capabilities | Configurações |

Reviewer 2: ```suggestion
| Produtos e capacidades | Configurações |
```

| --- | --- |

> | **Edge Application**<br></br>Application Acceleration<br></br>Edge Caching<br></br>Edge Functions<br></br>Image Processor<br></br>Load Balancer<br></br>L2 Caching | Gerenciar módulos <br></br>Gerenciar configurações relacionadas a módulos<br></br>Alterar configurações de edge application<br></br>Alterar configurações de cache<br></br>Alterar funções<br></br>Alterar respostas de erro<br></br>Alterar rules engine<br></br>Alterar origens |

Reviewer 2: Alterar regas ao invés de Alterar rules engine?

| **Digital Certificates** | Instalar certificados<br></br>Renovar certificados<br></br>Configurar e parametrizar exceções |
| **Real-Time Purge** | Purge de cache no edge<br></br>Purge de cache na L2 |
| **Domains** | Alterar configurações de domínio |

> | **Edge Firewall**<br></br>Network Layer Protection<br></br>Web Application Firewall<br></br>Network Lists | Alterar configurações de firewall<br></br>Alterar configurações de políticas WAF<br></br>Criar regras WAF personalizadas<br></br>Criar network lists personalizadas<br></br>Definir as configurações para geo blocking |

Reviewer: ```suggestion
| **Edge Firewall**<br></br>Network Layer Protection<br></br>Web Application Firewall<br></br>Network Lists | Alterar configurações de firewall<br></br>Alterar configurações de políticas WAF<br></br>Criar regras WAF personalizadas<br></br>Criar network lists personalizadas<br></br>Definir as configurações para bloqueio por geolocalização |
```

| **Intelligent DNS** | Criar zonas<br></br>Gerenciar registros | 
| **Edge Orchestrator**<br></br>Edge Services<br></br>Edge Nodes | Gerenciar edge services<br></br>Alterar configurações de edge nodes |
| **Data Streaming** | Criar data streamings<br></br>Gerenciar data streamings |
| **Real-Time Metrics** | Recuperar dados |
| **Real-Time Events** | Fornecer logs de dados |

> 
> ## Atividades fora do escopo

Reviewer: ```suggestion

---

## Atividades fora do escopo
```

> - **Suporte técnico**: todos os [tickets de suporte](/en/documentation/products/guides/open-tickets/) devem ser abertos através dos canais oficiais de suporte da Azion e serão atribuídos à equipe apropriada.

Reviewer 2: ```suggestion
- **Suporte técnico**: todos os [tickets de suporte](/pt-br/documentacao/produtos/guias/abrir-tickets/) devem ser abertos através dos canais oficiais de suporte da Azion e serão atribuídos à equipe apropriada.
```

> - **Suporte a outros provedores**: este serviço não oferece suporte a qualquer infraestrutura e aplicações que não estejam rodando na Azion ou integrações do Marketplace da Azion fornecidas por fornecedores terceiros.

Reviewer 2: fornecidas por fornecedores -- sera que achamos um sinônimo?

> - **Suporte a outros provedores**: este serviço não oferece suporte a qualquer infraestrutura e aplicações que não estejam rodando na Azion ou integrações do Marketplace da Azion fornecidas por fornecedores terceiros.

Reviewer: ```suggestion
- **Suporte a outros provedores**: este serviço não oferece suporte a qualquer infraestrutura e aplicações que não estejam rodando na Azion ou integrações do Marketplace da Azion fornecidas por terceiros.
```

> - **Mitigar falhas de workload ou ataques DDoS**: o [Security Response Team](/en/documentation/services/security-response-team/) provê o servico para mitigar falhas de workload failures e ataques DDoS.

Reviewer 2: ```suggestion
- **Mitigar falhas de workload ou ataques DDoS**: o [Security Response Team](/pt-br/documentacao/servicos/security-response-team/) provê o serviço para mitigar falhas de workload failures e ataques DDoS.
```

> - **Auditorias de segurança**: este serviço não realiza auditorias de segurança. A Azion fornece suporte de conformidade para produtos Azion, como SOC 2/3, PCI, FIPS 140 Nível 3 e Mapas Personalizados na Edge Network.

Reviewer: ```suggestion
- **Auditorias de segurança**: este serviço não realiza auditorias de segurança. A Azion fornece suporte de conformidade para produtos Azion, como **SOC 2/3**, **PCI**, **FIPS 140 Nível 3** e **Mapas Personalizados na Edge Network**.
```

- **Gerenciar dispositivos de usuários finais**: este serviço não gerencia dispositivos de usuários finais, como laptops, desktops ou smartphones.

> 

Reviewer: ```suggestion
```

| **Digital Certificates** | Instalar certificados<br></br>Renovar certificados<br></br>Configurar e parametrizar exceções |
| **Real-Time Purge** | Purge de cache no edge<br></br>Purge de cache na L2 |
| **Domains** | Alterar configurações de domínio |
| **Edge Firewall**<br></br>Network Layer Protection<br></br>Web Application Firewall<br></br>Network Lists | Alterar configurações de firewall<br></br>Alterar configurações de políticas WAF<br></br>Criar regras WAF personalizadas<br></br>Criar network lists personalizadas<br></br>Definir as configurações para geo blocking |
| **Intelligent DNS** | Criar zonas<br></br>Gerenciar registros | 
| **Edge Orchestrator**<br></br>Edge Services<br></br>Edge Nodes | Gerenciar edge services<br></br>Alterar configurações de edge nodes |
| **Data Streaming** | Criar data streamings<br></br>Gerenciar data streamings |
| **Real-Time Metrics** | Recuperar dados |
| **Real-Time Events** | Fornecer logs de dados |

## Atividades fora do escopo

- **Suporte técnico**: todos os [tickets de suporte](/en/documentation/products/guides/open-tickets/) devem ser abertos através dos canais oficiais de suporte da Azion e serão atribuídos à equipe apropriada.

> - **Suporte a outros provedores**: este serviço não oferece suporte a qualquer infraestrutura e aplicações que não estejam rodando na Azion ou integrações do Marketplace da Azion fornecidas por terceiros.

Author: ```suggestion
- **Suporte a outros provedores**: este serviço não oferece suporte a qualquer infraestrutura e aplicações que não estejam rodando na Azion ou através de integrações do Marketplace da Azion fornecidas por terceiros.
```

---
title: Managed Configurations

> description: Ao adquirir este serviço, uma equipe de engenheiros experientes pode realizar certas tarefas diárias em seu nome, como configurar, monitorar e manter as configurações para você.

Author: ```suggestion
description: Managed Configurations permite que uma equipe de engenheiros experientes possa realizar certas tarefas diárias em seu nome, como configurar, monitorar e manter as configurações.
```

> description: Managed Configurations permite que uma equipe de engenheiros experientes possa realizar certas tarefas diárias em seu nome, como configurar, monitorar e manter as configurações.

Author: ```suggestion
description: Managed Configurations permite que uma equipe de engenheiros experientes possa realizar certas tarefas diárias em seu nome, como configurar e manter suas configurações.
```

meta_tags: serviços, Managed Configurations, configurações
namespace: documentation_services_managed_configuration
permalink: /documentacao/servicos/managed-configurations/
---

> O serviço de **Managed Configurations** oferece gerenciamento e controle sobre as configurações da sua infraestrutura edge na Azion. Este serviço permite que uma equipe de engenheiros experientes possa realizar certas tarefas diárias em seu nome, como configurar e manter suas configurações.

Author: ```suggestion
O serviço de **Managed Configurations** oferece gerenciamento e controle sobre a sua infraestrutura edge na Azion. Este serviço permite que uma equipe de engenheiros experientes possa realizar certas tarefas diárias em seu nome, como configurar e manter suas configurações.
```

> O serviço de **Managed Configurations** oferece gerenciamento e controle sobre as configurações da sua infraestrutura edge na Azion. Ao adquirir este serviço, uma equipe de engenheiros experientes pode realizar certas tarefas diárias em seu nome, como configurar, monitorar e manter as configurações.

Author: ```suggestion
O serviço de **Managed Configurations** oferece gerenciamento e controle sobre as configurações da sua infraestrutura edge na Azion. Este serviço permite que uma equipe de engenheiros experientes possa realizar certas tarefas diárias em seu nome, como configurar, monitorar e manter suas configurações.
```

> O serviço de **Managed Configurations** oferece gerenciamento e controle sobre as configurações da sua infraestrutura edge na Azion. Este serviço permite que uma equipe de engenheiros experientes possa realizar certas tarefas diárias em seu nome, como configurar, monitorar e manter suas configurações.

Author: ```suggestion
O serviço de **Managed Configurations** oferece gerenciamento e controle sobre as configurações da sua infraestrutura edge na Azion. Este serviço permite que uma equipe de engenheiros experientes possa realizar certas tarefas diárias em seu nome, como configurar e manter suas configurações.
```

> O serviço de Managed Configurations é operado por arquitetos de soluções da Azion, que são responsáveis por implementar as alterações de configuração solicitadas por clientes. Para solicitar o serviço de Managed Configurations para aplicações rodando na Azion, [abra um Ticket de Suporte](https://tickets.azion.com/).

Author: ```suggestion
O serviço de Managed Configurations é operado por arquitetos de soluções da Azion, que são responsáveis por implementar as alterações de configuração solicitadas por você como cliente Azion. Para solicitar o serviço de Managed Configurations para aplicações rodando na Azion, [abra um Ticket de Suporte](https://tickets.azion.com/).
```

> O serviço de Managed Configurations é operado pela equipe de Arquitetos de Soluções da Azion, que será responsável por implementar as alterações de configuração solicitadas pelo cliente da Azion. Para solicitar o serviço de Managed Configurations para aplicações rodando na Azion, [abra um Ticket de Suporte](https://tickets.azion.com/).

Author: ```suggestion
O serviço de Managed Configurations é operado por arquitetos de soluções da Azion, que são responsáveis por implementar as alterações de configuração solicitadas por clientes. Para solicitar o serviço de Managed Configurations para aplicações rodando na Azion, [abra um Ticket de Suporte](https://tickets.azion.com/).
```

:::note

> Este add-on está disponível para os Planos de Serviço **Enterprise** e **Mission Critical**. Este serviço tem um pacote mensal com um preço fixo, limitado por solicitação de configuração. Cada solicitação do serviço dá direito a uma alteração de configuração. Consulte [Preços](https://www.azion.com/pt-br/precos/) para mais informações sobre como este serviço é cobrado.

Author: ```suggestion
Este add-on está disponível para os Planos de Serviço **Enterprise** e **Mission Critical**. Este serviço tem um pacote mensal com um preço fixo, limitado por solicitações de configuração. Cada solicitação do serviço dá direito a uma alteração de configuração. Consulte [Preços](https://www.azion.com/pt-br/precos/) para mais informações sobre como este serviço é cobrado.
```

:::

---

## Escopo

A tabela abaixo descreve os produtos e funcionalidades inclusos, bem como as principais configurações no escopo das **Managed Configurations**.

| Produtos e funcionalidades | Configurações |
| --- | --- |

> | **Edge Application**<br></br>Application Acceleration<br></br>Edge Caching<br></br>Edge Functions<br></br>Image Processor<br></br>Load Balancer<br></br>L2 Caching | Gerenciar módulos <br></br>Gerenciar configurações relacionadas a módulos<br></br>Alterar configurações de edge application<br></br>Alterar configurações de cache<br></br>Alterar funções<br></br>Alterar respostas de erro<br></br>Alterar rules engine<br></br>Alterar origens |

Author: ```suggestion
| **Edge Application**<br></br>Application Acceleration<br></br>Edge Caching<br></br>Edge Functions<br></br>Image Processor<br></br>Load Balancer<br></br>L2 Caching | Gerenciar módulos <br></br>Gerenciar configurações relacionadas a módulos<br></br>Alterar configurações de edge application<br></br>Alterar configurações de cache<br></br>Alterar funções<br></br>Alterar respostas de erro<br></br>Alterar regras<br></br>Alterar origens |
```

| **Digital Certificates** | Instalar certificados<br></br>Renovar certificados<br></br>Configurar e parametrizar exceções |
| **Real-Time Purge** | Purge de cache no edge<br></br>Purge de cache na L2 |
| **Domains** | Alterar configurações de domínio |
| **Edge Firewall**<br></br>Network Layer Protection<br></br>Web Application Firewall<br></br>Network Lists | Alterar configurações de firewall<br></br>Alterar configurações de políticas WAF<br></br>Criar regras WAF personalizadas<br></br>Criar network lists personalizadas<br></br>Definir as configurações para bloqueio por geolocalização |
| **Intelligent DNS** | Criar zonas<br></br>Gerenciar registros | 
| **Edge Orchestrator**<br></br>Edge Services<br></br>Edge Nodes | Gerenciar edge services<br></br>Alterar configurações de edge nodes |
| **Data Streaming** | Criar data streamings<br></br>Gerenciar data streamings |

> | **Real-Time Metrics** | Recuperar dados |

Reviewer 2: ```suggestion
| **Real-Time Metrics** | Buscar dados |
```
